.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "mocha": true,
```

Comment:
I'm not sure why mocha was set to true in our jshintrc, but it doesn't need to be here since we don't use mocha.